drivers/memc/memc_stm32_sdram.c

```diff
@@ -129,5 +129,7 @@ static const struct memc_stm32_sdram_config config = {
 	.banks_len = ARRAY_SIZE(bank_config),
 };
 
+#if DT_NODE_HAS_STATUS(DT_INST(0, st_stm32_fmc_sdram), okay)
```

Comment:
In this case it would not be needed because Kconfig option selecting SDRAM already takes into account DT status, but it can still be added.

```suggestion
#if DT_NODE_HAS_STATUS(DT_DRV_INST(0), okay)
```